app/src/main/kotlin/com/intive/picover/main/navigation/view/PicoverNavHost.kt

```diff
@@ -70,9 +62,49 @@ fun PicoverNavHost(
 			) {
 				PartyDetailsScreen(hiltViewModel())
 			}
+			profileGraph(navController)
 			composable("articles") {
 				ArticlesScreen(hiltViewModel())
 			}
 		}
 	}
 }
+
+@OptIn(ExperimentalMaterialNavigationApi::class)
+private fun NavGraphBuilder.profileGraph(navController: NavHostController) {
+	navigation(startDestination = "profile", route = "profileGraph") {
+		composable("profile") {
+			val parentEntry = it.rememberParentEntry(navController)
+			ProfileScreen(hiltViewModel(parentEntry), navController)
+		}
+		dialog("deleteAccount") {
+			val viewModel: ProfileViewModel = hiltViewModel()
```

Comment:
`deleteAccount` should have separate view model. Anyone willing to extract related logic from `ProfileViewModel`?

Reply:
I can handle it

Reply:
https://github.com/Android-Guild/Picover/issues/397